plan/common_plans.go

```diff
@@ -407,51 +406,97 @@ type Explain struct {
 	explainedPlans map[int]bool
 }
 
-// prepareExplainInfo4DAGTask generates the following information for every plan:
-// ["id", "parents", "task", "operator info"].
-func (e *Explain) prepareExplainInfo4DAGTask(p PhysicalPlan, taskType string, parentID string) {
-	childrenIDs := make([]string, 0, len(p.Children()))
-	for _, ch := range p.Children() {
-		childrenIDs = append(childrenIDs, ch.ExplainID())
+const (
```

Comment:
Since this change is large enough, what about just remove these things to `explain.go`

Reply:
It's better to be remained on this file for now. `explain.go` is for operators to print their explain info. we can do these code clean work in another pr.